Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM node:current as builder
 
 WORKDIR /tmp/cirrus-ci-web
-ADD package.json yarn.lock /tmp/cirrus-ci-web/
+# add dependency data files
+ADD package.json yarn.lock .yarnrc.yml /tmp/cirrus-ci-web/
+# add relevant files from .yarn
+ADD .yarn /tmp/cirrus-ci-web/.yarn
```

Comment:
I think for more effective layer caching, this line should be before adding all the `package.json yarn.lock .yarnrc.yml` files.

Reply:
I tried that, but kept getting strange errors, for instance it was copying .yarn/* into rootDir instead of rootDir/.yarn/*